app/components/table-row.js

```diff
@@ -142,13 +142,17 @@ const Row = ({
   const { writable, metadata, key } = dat
   const { title } = metadata
   const placeholderTitle = `#${key}`
-  return (<Tr onClick={() => inspectDat(dat.key)}>
```

Comment:
How does that change come by? Why is it part of this PR?

Reply:
The linter does that.